go-controller/pkg/ovn/admin_network_policy_test.go

```diff
@@ -299,7 +299,7 @@ var _ = ginkgo.Describe("OVN ANP Operations", func() {
 		app.Name = "test"
 		app.Flags = config.Flags
 
-		fakeOVN = NewFakeOVN(false)
+		fakeOVN = NewFakeOVN(false, nil)
```

Comment:
How about we create a *new* function for this, instead of force adding a parameter to the current one, which forces you to change a lot of files ?

Reply:
yep thats the plan